Description: Relax version requirements of mysql2 and factory_girls
Author: Balasankar C <balasankarc@autistici.org>
Last-Update: 2016-04-06
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Gemfile
+++ b/Gemfile
@@ -29,7 +29,7 @@
 end
 
 # Support libs
-gem "factory_girl", "~> 4.2.0"
+gem "factory_girl", "~> 4.2"
 gem "timecop"
 gem "chronic"
 gem "mocha", "~> 1.3.0"
